README.md

```diff
@@ -598,3 +599,51 @@ tugboat:php:init:
     - apt-get install -y libldap2-dev
     - docker-php-ext-install ldap
 ```
+
+Creating comprehensive documentation for conducting peer reviews on automated updates within the "Drainpipe" project is crucial to ensure consistency, maintainability, and the quality of the codebase. Here's a draft outline and content you could use or adapt for your documentation:
+
+## Peer Review Guidelines for Automated Updates
+
+These are guidelines for conducting peer reviews on automated dependency update pull requests created by Renovate. 
+
+## Automated Testing with GitHub Actions
+
+### Overview
+
+All automated updates submitted by Renovate undergo a series of automated tests via GitHub Actions. These tests are designed to ensure compatibility and stability with the new versions of dependencies.
+
+## Semantic Versioning and Update Types
+
+Drainpipe adheres to semantic versioning, categorizing updates into patches, minor, and major updates. Understanding these categories is crucial for the review process:
+
+- **Patch Releases (`x.y.Z`)**: Updates that fix bugs without adding new features or breaking existing functionality. If all tests pass, these updates are generally safe to merge.
+- **Minor Point Releases (`x.Y.z`)**: Updates that introduce new features without breaking backward compatibility. While these are often safe to merge if all tests pass, they require more scrutiny:
+  - Read the change logs carefully to understand the new features and fixes.
+  - Assess if the changes necessitate additional test coverage or could potentially impact existing functionality.
+  - Consider the implications of new features on the project's future development and maintenance.
```

Comment:
I think this should go in a separate section as it's not really related to us internally accepting Renovate PRs

Reply:
Maybe it's worded wrong? The point I was attempting to make here is that there are different levels of testing based on the type of update are being made. 

What if we just re-worded this to say 
```
All Renovate peer reviews regardless if they're a minor or patch release require:
- Reading the change logs carefully to understand the new features and fixes.
  - Assess if the changes necessitate additional test coverage or could potentially impact existing functionality.
  - Consider the implications of new features on the project's future development and maintenance.
- All tests and checks must pass
```

Reply:
I've updated this section with what I had suggested above. 

This is ready for review.